app/routes/public/exhibition/view.js

```diff
@@ -12,6 +12,9 @@ export default class ViewRoute extends Route {
     const event = this.modelFor('public');
     return hash({
       event,
+      exhibitors: event.query('exhibitors', {
```

Comment:
We can't load exhibitors here for the reasons discussed in the meeting. Secondly, it only loads 20 exhibitors, not all

Reply:
OK, so how will all the exhibitors be loaded at once?
This gives back 20.
```
  async beforeModel() {
    const event = this.modelFor('public');
    this.exhibitors = await event.query('exhibitors', {
      sort: 'position'
    });
  }
```
Also I am confused as beforeModel loads first, so shouldn't this be done in afterModel?

Reply:
Exactly, it should be done in afterModel. Use `'page[size]': 0`